### PR TITLE
PS-5.7-7035 Release Notes for PS-5.7.30-33

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.6 release notes
modified conf.py
modified release-notes_index
added Percona-Server-5.7.30-33

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+* :jirabug:`PS-6979`: Modified processing to call clean up functions as part of event (Upstream :mysqlbug:`99200`)
+* :jirabug:`PS-6860`: The innodb_buffer_pool_pages_LRU_flushed variable did not work.
+* :jirabug:`PS-6811`: Service failed to start asserting ACL_PROXY_USER::check_validity if --skip-name-resolve=1 and there is a Proxy user. (Upstream :mysqlbug:`98908`)
+* :jirabug:`PS-6112`: Inconsistent Binlog_snapshot_gtid when mysqldump was used with --single-transaction.
+* :jirabug:`PS-6945`: The tokubackup was broken since it no longer exported the correct large file API.
+* :jirabug:`PS-6856`: corrupted binlogs in PS 5.7.28 and 5.7.29 (Upstream :mysqlbug:`97531`)
+* :jirabug:`PS-6946`: Sanitizers found memory use after free and other issues in tokubackup.
+* :jirabug:`PS-5893`: Added support for running multiple instances with systemD on Debian.
+* :jirabug:`PS-4573`: The 5.7 patch created a mysqld.cnf file which caused other configurations from functioning correctly.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.7.30-33.rst]: This doesn't make much sense to me, I can't tell what was broken or what was fixed.  The original bug description was much more clear as to what the problem is/was.  I had to look at the JIRA bug too understand even that this is a documentation change and not an actual bug.  Please try to reword this somehow.

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6979`: Modified processing to call clean up functions as part of event (Upstream :mysqlbug:`99200`)
+* :jirabug:`PS-6860`: The innodb_buffer_pool_pages_LRU_flushed variable did not work.
+* :jirabug:`PS-6811`: Service failed to start asserting ACL_PROXY_USER::check_validity if --skip-name-resolve=1 and there is a Proxy user. (Upstream :mysqlbug:`98908`)
+* :jirabug:`PS-6112`: Inconsistent Binlog_snapshot_gtid when mysqldump was used with --single-transaction.
+* :jirabug:`PS-6945`: The tokubackup was broken since it no longer exported the correct large file API.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.7.30-33.rst]: tokubackup plugin

**Review by george-lorch — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+* :jirabug:`PS-6902`: Documented the new variable enable_insert_with_update_caching.
+* :jirabug:`PS-4464`: Exposed the last global transaction identifier (GTID) executed for a CONSISTENT SNAPSHOT.
+* :jirabug:`PS-6886`: Documented a new variable rocksdb_cache_dump which includes RocksDB block cache content in a core dump.
+* :jirabug:`PS-6890`: Documented the new variable blind_delete_primary_key
+* :jirabug:`PS-6885`: Documented the new variable rocksdb_rollback_on_timeout which allows the rollback of an entire transaction on timeout.
+* :jirabug:`PS-6910`: Documented the new variable rocksdb_stats_level.
+* :jirabug:`PS-6901`: Documented the new variable read_free_rpl
+* :jirabug:`PS-6891`: Documented the new variable master_skip_tx_api
+
```
Comment (inikep) [inline doc/source/release-notes/Percona-Server-5.7.30-33.rst:29]: These DOC tickets are missing:
https://jira.percona.com/browse/PS-6926
https://jira.percona.com/browse/PS-6951

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-6984`: Updated the zstd submodule.
```
Comment (inikep) [inline doc/source/release-notes/Percona-Server-5.7.30-33.rst]: Updated the zstd submodule to v1.4.4

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+* :jirabug:`PS-6979`: Modified processing to call clean up functions as part of event (Upstream :mysqlbug:`99200`)
+* :jirabug:`PS-6860`: The innodb_buffer_pool_pages_LRU_flushed variable did not work.
+* :jirabug:`PS-6811`: Service failed to start asserting ACL_PROXY_USER::check_validity if --skip-name-resolve=1 and there is a Proxy user. (Upstream :mysqlbug:`98908`)
+* :jirabug:`PS-6112`: Inconsistent Binlog_snapshot_gtid when mysqldump was used with --single-transaction.
+* :jirabug:`PS-6945`: The tokubackup was broken since it no longer exported the correct large file API.
+* :jirabug:`PS-6856`: corrupted binlogs in PS 5.7.28 and 5.7.29 (Upstream :mysqlbug:`97531`)
+* :jirabug:`PS-6946`: Sanitizers found memory use after free and other issues in tokubackup.
+* :jirabug:`PS-5893`: Added support for running multiple instances with systemD on Debian.
+* :jirabug:`PS-4573`: The 5.7 patch created a mysqld.cnf file which caused other configurations from functioning correctly.
+* :jirabug:`PS-7020`: Converted MTR tests from python2 to python3.
```
Comment (inikep) [inline doc/source/release-notes/Percona-Server-5.7.30-33.rst]: In Ubuntu 20.04, only python3-mysqldb is available. We converted the MTR tests to support both Python2 and Python3. The scripts require Python 2.6 or newer.

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.30-33.rst]: Will fix. Thanks.

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.30-33.rst]: Will add.

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.30-33.rst:29]: Will check.

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.30-33.rst]: Thanks. Will fix.

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.30-33.rst:29]: Added.

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.7.30-33.rst [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+* :jirabug:`PS-7031`: add gcc-10 to platforms tested by Travis and Azure
+* :jirabug:`PS-6984`: Updated the zstd submodule to v1.4.4.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6979`: If CREATE USER remains in the processlist, other users may see it. Modified the processing to call clean up functions as part of event. (Upstream :mysqlbug:`99200`)
```
Comment (altmannmarcelo) [inline doc/source/release-notes/Percona-Server-5.7.30-33.rst]:
```suggestion
* :jirabug:`PS-6979`: CREATE USER statement could remain in the processlist after the statement has completed, causing other users to may see it. Modified the processing to call clean up functions as part of event cleanup. (Upstream :mysqlbug:`99200`)
```

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.30-33.rst]: Changed to:
The CREATE USER statement could remain in the processlist after the statement has completed, allowing other users to may see it. Modified the processing to call clean up functions as part of event cleanup.

**Comment by inikep** [comment]
@georgelorchpercona We mention introduction of many MyRocks params. Shouldn't we mention that MyRocks was updated up to `fb-prod201907`?

**Review by altmannmarcelo — APPROVED** [review]

**Review by kathy-williamson — APPROVED** [review]

**Pull request merged**